### ISSUE TITLE
Updating picklist from another database

### ISSUE THREAD

**Issue by ghuotvez** [NRCan/GSC-Field-Application#89]
From @saralizj

> Today I tried using the functionality to load picklists, with no luck.
> Here’s what I did:
> -	Edited a picklist, as a test, on my laptop in the GSC app
> -	I thought the download button would allow me to download the custom lists, but it seems only to have load functionality?
> -	So I backed up my entire field notebook and transferred the files to my tablet;
> -	Pressing load button on tablet crashes program.  Tried several times.

**Comment by ghuotvez** [comment]
Will need to follow up I can't reproduce the problem. I did the previous sequence, which is to backup the whole field book and I tried tried importing the picklist from within the settings, picklist tab. It's working good. Maybe the changes from the previous closed issues #84, #49 fixed something in the code. I also took the time to update the label, shortening it so it fits on the burgundy bar when we click on the ellipsis.
